docs/develop/Data_Format.md

```diff
@@ -1,13 +1,15 @@
 # Data Format
 
-!!! warning "Work in Progress"
-    This is a work in progress. Feel free to contribute.
+ElectionGuard expects data to be formatted in a particular way. The expectation is to keep this format versioned alongside the specification. The data format will be defined here to indicate to developers what they should be expecting. 
 
+Keep in mind that not all serialized files should be public. The [election record][election-record] includes the list of all data formats that should be public. [Sample data][sample-data] for the election is also available for developers. This will include both what is in the Election Record, but also some private data only available for testing. 
 
 ## Schema ![Version 0.95.0][shield-green-0.95.0]
 
-These are [JSON schema][json-schema] files that can be used to verify the schema of the setup files like the manifest pre-election and the election record files post-election.
+Currently, JSON is used as the primary serialization of the artifacts from an election. These are [JSON schema][json-schema] files that can be used to verify the schema of the setup files like the manifest pre-election and the election record files post-election. This represents the full list of items that can be serialized / deserialized, but not all elections will use every file type. 
```

Comment:
`primary serialization` -> `primary serialization method`

Reply:
I would say primary serialization format since JSON itself is not a method.